Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -71,6 +74,9 @@ def download_data(cls, data_dir, prompt=True):
                 print('Download cancelled')
                 return False
 
+            if zenodo_get is None:
+                raise ImportError(f"Please install optional dependency zenodo_get to use dataexample.REMOTEDATA.download_data")
+            
```

Comment:
here you can add the try/except if you want to be kind with the user.
```suggestion
            try:
                from zenodo_get import zenodo_get
            except ImportError as ie:
                raise ImportError(f"Please install optional dependency zenodo_get to use dataexample.REMOTEDATA.download_data")
            
```

Unittests will/should not fail at this line as they would not run if `zenodo_get` is not installed (hopefully).

Reply:
Actually I think I figured out how to fix the tests now so we can do this